### PR TITLE
Navigation (p2)

### PR DESCRIPTION
**Pull request by pradipgharge**
added navigation with hamburger menu.

### CODE REVIEW

File: pages/components/Navigation/navigation.css [hunk 1]
```diff
@@ -12,71 +12,139 @@
     padding: 1rem;
 }
 
-/* Simple navbar styling */
-.navbar-basic{
+.navbar-container {
+    height: 15rem;
+    width: 100%;
+    overflow-y: auto;
+    border: 1px solid black;
```
Comment (Ankur9669) [inline pages/components/Navigation/navigation.css]: Suggestion: Can use css variable

**Review by Ankur9669 — APPROVED** [review]
The Navigation transiotion is looking awesome great work👏

File: pages/components/Navigation/navigation.css [hunk 2]
```diff
@@ -12,71 +12,139 @@
...
     display: flex;
     justify-content: center;
     border: none;
     background: none;
 }
 
-.nav-input{
+.searchbar-input {
     border: none;
     outline: none;
```
Comment (yaswanthmyneni) [inline pages/components/Navigation/navigation.css]: changes: give outline: 0 instead of none

File: pages/components/Navigation/navigation.css [hunk 3]
```diff
@@ -12,71 +12,139 @@
...
+
+
+/* Hamburger Styling */
+.drawer-container {
+    padding: 1rem;
+    width: 12rem;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    left: -200px;
```
Comment (yaswanthmyneni) [inline pages/components/Navigation/navigation.css]: changes: give rem greater than 15px

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes

**Pull request merged**